src/accordion/accordion.ts

```diff
@@ -175,6 +193,16 @@ export class NgbAccordion implements AfterContentChecked {
       this._panelRefs.set(panel.id, panel);
     });
   }
+
+  private _setClassType() {
```

Comment:
` [class]="'card-header card-' + (panel.type ? panel.type: type)"`

i thought of this.